javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -1704,6 +1704,97 @@ public static boolean sameTree(ExpressionTree expr1, ExpressionTree expr2) {
     }
   }
 
+  /**
+   * Returns the selector expression of {@code switchExpressionTree}. For example
+   *
+   * <pre>
+   *   switch ( <em>expression</em> ) { ... }
+   * </pre>
+   *
+   * @param switchExpressionTree the switch expression whose selector expression is returned
+   * @return the selector expression of {@code switchExpressionTree}
+   */
+  public static ExpressionTree switchExpressionTreeGetExpression(Tree switchExpressionTree) {
```

Comment:
Should this be named "switchExpressionTreeGet**Selector**Expression" or "switchExpressionTreeGet**Selector**"?  You could drop the "Tree" from the name to save 4 characters, but don't do that if it would introduce inconsistency.

Reply:
So far we have named the methods that reflectively call a Java 17 method `classNameMethodName`.  I think it is easier to find the utility correct utility method if it follows this pattern.